mtoken-model/src/main/java/com/wultra/security/powerauth/lib/mtoken/model/enumeration/ErrorCode.java

```diff
@@ -42,6 +42,18 @@ public class ErrorCode {
      */
     public static final String INVALID_ACTIVATION           = "INVALID_ACTIVATION";
 
+    /**
+     * Error code for situation when an invalid application identifier is
+     * attempted for operation manipulation.
+     */
+    public static final String INVALID_APPLICATION          = "INVALID_APPLICATION";
```

Comment:
I suppose that these error codes should be discussed with @kober32 and @Hopsaheysa

Reply:
It would be nice to give us examples of when those 2 errors can happen so we can make a hint for the mobile devs what to show to the user.

Reply:
This controller handles endpoints on the path `/api/auth/token/app`

**INVALID_APPLICATION** (status 400) happens if the application specified by `applicationId` in a request does not exist. Relevant to:
```
/operation/list
/operation/history
/operation/authorize
/operation/cancel
```

**INVALID_OPERATION** (status 400) may happen in any of the three cases:  
1. Operation specified by `operationId` in a request does not exist. Relevant to:
```
/operation/detail
/operation/detail/claim
/operation/authorize
/operation/cancel
```
2. Operation is not claimed to the user specified by `userId` in a request. Relevant to:
```
/operation/authorize
/operation/cancel 
```
3. Operation specified by `operationId` in a request is not in the `PENDING` state. Relevant to:
```
/operation/authorize
/operation/cancel 
```

**INVALID_REQUEST** (status 400) may happen in any of the six endpoints in case a required field of a request is missing.

**REMOTE_COMMUNICATION_ERROR** (status 409) may happen in any of the six endpoints in case an unexpected error caused by remote server occurs.